[v3.0.0-beta.2] Type `Tab.Pane` is missing

## Bug Report

`Tab.Pane` type is missing in the TypeScript type definition from v3.0.0-beta.2.

It seems the property was mistakenly dropped from Tab.d.ts in this commit: https://github.com/Semantic-Org/Semantic-UI-React/commit/e243fa47ac28b4f67141c73a3a0141afe9317b8a#diff-a4fd743c7d3d12205492c1129d5fc47d15c620f7a76a38a1de0864f18e605092

### Steps
Type-check the following code in TypeScript.

```tsx
import { Tab } from 'semantic-ui-react'
export const App = () => <Tab panes={
  [{
    menuItem: { content: 'tab' },
    render: () => <Tab.Pane>pane</Tab.Pane>
  }]
} />
```

### Expected Result
It's well-typed.

### Actual Result
```
src/Test.tsx:5:24 - error TS2339: Property 'Pane' does not exist on type 'ForwardRefComponent<TabProps, HTMLDivElement>'.

5     render: () => <Tab.Pane>pane</Tab.Pane>
                         ~~~~

src/Test.tsx:5:39 - error TS2339: Property 'Pane' does not exist on type 'ForwardRefComponent<TabProps, HTMLDivElement>'.

5     render: () => <Tab.Pane>pane</Tab.Pane>
                                        ~~~~

```

### Version
3.0.0-beta.2

### Testcase

--- a/src/modules/Tab/Tab.d.ts
+++ b/src/modules/Tab/Tab.d.ts
@@ -1,7 +1,7 @@
 import * as React from 'react'
 
 import { ForwardRefComponent, SemanticShorthandItem } from '../../generic'
-import { TabPaneProps } from './TabPane'
+import TabPane, { TabPaneProps } from './TabPane'
 
 export interface TabProps extends StrictTabProps {
   [key: string]: any
@@ -58,6 +58,8 @@ export interface StrictTabProps {
   renderActiveOnly?: boolean
 }
 
-declare const Tab: ForwardRefComponent<TabProps, HTMLDivElement>
+declare const Tab: ForwardRefComponent<TabProps, HTMLDivElement> & {
+  Pane: typeof TabPane
+}
 
 export default Tab

